install/ontology/blueprints.rdf

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<rdf:RDF
+        xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+        xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+        xml:base="http://www.taotesting.com/ontologies/blueprints.rdf#"
+        xmlns:widget="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#"
+        xmlns:generis="http://www.tao.lu/Ontologies/generis.rdf#"
+        xmlns:tao="http://www.tao.lu/Ontologies/TAO.rdf#"
+>
+    <!-- Class to represent test blueprints -->
+    <!-- Contains identifer, title & content to store additional info -->
+    <rdf:Description rdf:about="http://www.taotesting.com/ontologies/blueprints.rdf#Blueprints">
```

Comment:
Could you please rename the class name into Blueprint (without a 's') ? I know it's picky but, I know you like super nice code ;-) !